server/spinwick.go

```diff
@@ -792,6 +815,36 @@ func (s *Server) destroyKubeSpinWick(pr *model.PullRequest) *spinwick.Request {
 	request.InstallationID = namespaceName
 	mlog.Info("Kube namespace " + namespaceName + " has been destroyed")
 
+	headers := map[string]string{
+		"x-api-key": s.Config.AWSAPIKey,
+	}
+	cloudClient := cloudModel.NewClientWithHeaders(s.Config.ProvisionerServer, headers)
+	webhooks, err := cloudClient.GetWebhooks(&cloudModel.GetWebhooksRequest{
+		OwnerID: namespaceName,
+	})
+	if err != nil {
+		mlog.Error("Failed to get webhooks for spinwick", mlog.Err(err))
+		request.Error = err
+		return request
+	}
+
+	for _, webhook := range webhooks {
```

Comment:
If I am not mistaken, there should only ever be 1 webhook returned here to be deleted right? If that's the case maybe we want to check for that first and return an error if we ever find more than that?

Reply:
Yeah that's the intention - this is mostly just a fallback in case it isn't for whatever reason (the CWS is doing the same thing as part of the DeleteStripeWebhook call)

The main reason I opted to just clean up rather than error is the general audience for the Spinwicks are QA and others on the team who wouldn't really have access to the Cloud CLI/Provisioner to manually go in and clean it up

Reply:
Yeah, I was mostly thinking about cases where maybe webhooks were returned that were outside of the scope of cleaning up the spinwick, but that really shouldn't happen so this should be fine.